kaitaistruct.py

```diff
@@ -14,8 +17,41 @@
 #
 __version__ = '0.9'
 
+class _KaitaiFromCtor(object):
+    """Adds to class methods to construct it from ctor"""
+    @classmethod
+    def from_file(cls, file, map=True):
+        if isinstance(file, str):
+            return cls.from_file(Path(file), map)
+        elif isinstance(file, Path):
+            with file.open("rb") as f:
+                return cls.from_file(f, map)
+        elif isinstance(file, _IOBase):
+            if map:
+                with mmap.mmap(file.fileno(), 0, access=mmap.ACCESS_READ) as buf:
+                    return cls.from_bytes(buf)
+            else:
+                return cls.from_io(file)
+        else:
+            raise Exception("file argument must be either file path (either str or Path) or a file object")
+
+    @classmethod
+    def from_any(cls, data, map=True):
+        """Parses a Kaitai Struct type"""
```

Comment:
This docstring will be misleading if found in KaitaiStream class. `KaitaiStream.from_any` would not "parse a Kaitai Struct type" — it will merely create a wrapped stream.

Reply:
Should I just remove it?